tests/components/accessibilityControls-test.js

```diff
@@ -117,8 +117,25 @@ describe('AccessibilityControls', function () {
     };
 
     var accessibilityControls = new AccessibilityControls(controllerMock);
-    accessibilityControls.handleKey(mockEvent);
-    accessibilityControls.handleKey(mockEvent);
+    accessibilityControls.keyEventDown(mockEvent);
+    accessibilityControls.keyEventDown(mockEvent);
+  });
+
+  it('tests "A" key', function () {
+    var controllerMock = {
+      videoVr: true,
+      state: {
+        accessibilityControlsEnabled: true,
+      },
+      moveToDirection: function() {},
+    };
+
+    var mockEvent = {
+      keyCode: 65,
+    };
+
+    var accessibilityControls = new AccessibilityControls(controllerMock);
+    accessibilityControls.keyEventDown(mockEvent);
```

Comment:
Would recommend testing specific behavior rather than just executing the method.

Reply:
ok, but it looks like it is enough in this situation.